controllers/users.controller.js

```diff
@@ -0,0 +1,13 @@
+const {getUserData} = require("../models/users.models")
+
+
+function getUsers(req, res, next) {
+    getUserData().then((users) => {
```

Comment:
Consider using synonyms of "get" here such as "select" or "fetch" to make sure that the controller function and model function are different and fall in line with convention.